src/catsDb.js

```diff
@@ -1,64 +1,81 @@
-
 // class 'CatRepository' serves as a template for creating new cat objects with props & methods
 
-const { splice } = require("./breedsDb");
-
 class CatRepository {
     constructor() {
         this.cats = [ // array of objects - if object, change to >> this.cats:{ 1:{}, 2:{}, 3:{}}
             {
                 id: 1,
                 name: "Catty",
-                sex: "F",
-                coat: "medium hair",
+                ageInYears: 1,
+                favouriteToy: "grass",
                 description: "buff or tan, skinny, talkative, often found in tall grasses or deep in the bush",
                 breedId: 1
             },
             {
                 id: 2,
                 name: "Frank",
-                sex: "M",
-                coat: "medium hair",
+                ageInYears: 5,
+                favouriteToy: "flies",
                 description: "orange, heavy-set, non-responsive except to neck scratches",
                 breedId: 1
             },
             {
                 id: 3,
                 name: "Pancake",
-                sex: "M",
-                coat: "shorthair",
+                ageInYears: 4,
+                favouriteToy: "pavement",
                 description: "brown, heavy-set, spent life behind bars",
                 breedId: 1
             },
             {
                 id: 4,
                 name: "Madame Floof",
-                sex: "F",
-                coat: "long hair",
+                ageInYears: 1,
+                favouriteToy: "ball",
                 description: "white dark patches on legs, kerbside, fluffy dustbuster tail",
                 breedId: 2
             },
         ];
     }
 
-    get getAllCats() {
+    get getCats() {
         return this.cats // returns [{},{},{}]
         }
     
     set setCats (catsCopy) {
         this.cats = catsCopy; 
     }
 
+
+    getAllCats() {
```

Comment:
would it be possible to use `map()` rather than pushing to a new array? this might be more concise

Reply:
Yes, you are clever! Although I was very excited to use a Factory Function.